cmd/podman/images.go

```diff
@@ -83,108 +84,79 @@ func (a imagesSortedSize) Less(i, j int) bool {
 }
 
 var (
-	imagesFlags = []cli.Flag{
-		cli.BoolFlag{
-			Name:  "all, a",
-			Usage: "Show all images (default hides intermediate images)",
-		},
-		cli.BoolFlag{
-			Name:  "digests",
-			Usage: "Show digests",
-		},
-		cli.StringSliceFlag{
-			Name:  "filter, f",
-			Usage: "Filter output based on conditions provided (default [])",
-		},
-		cli.StringFlag{
-			Name:  "format",
-			Usage: "Change the output format to JSON or a Go template",
-		},
-		cli.BoolFlag{
-			Name:  "noheading, n",
-			Usage: "Do not print column headings",
-		},
-		cli.BoolFlag{
-			Name:  "no-trunc, notruncate",
-			Usage: "Do not truncate output",
-		},
-		cli.BoolFlag{
-			Name:  "quiet, q",
-			Usage: "Display only image IDs",
-		},
-		cli.StringFlag{
-			Name:  "sort",
-			Usage: "Sort by created, id, repository, size, or tag",
-			Value: "Created",
-		},
-	}
-
+	imagesCommand     cliconfig.ImagesValues
 	imagesDescription = "lists locally stored images."
-	imagesCommand     = cli.Command{
-		Name:                   "images",
-		Usage:                  "List images in local storage",
-		Description:            imagesDescription,
-		Flags:                  sortFlags(imagesFlags),
-		Action:                 imagesCmd,
-		ArgsUsage:              "",
-		UseShortOptionHandling: true,
-		OnUsageError:           usageErrorHandler,
-	}
-	lsImagesCommand = cli.Command{
-		Name:                   "list",
-		Aliases:                []string{"ls"},
-		Usage:                  "List images in local storage",
-		Description:            imagesDescription,
-		Flags:                  imagesFlags,
-		Action:                 imagesCmd,
-		ArgsUsage:              "",
-		UseShortOptionHandling: true,
-		OnUsageError:           usageErrorHandler,
+
+	_imagesCommand = &cobra.Command{
+		Use:   "images",
+		Short: "List images in local storage",
+		Long:  imagesDescription,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			imagesCommand.InputArgs = args
+			imagesCommand.GlobalFlags = MainGlobalOpts
+			return imagesCmd(&imagesCommand)
+		},
+		Example: "",
 	}
 )
 
-func imagesCmd(c *cli.Context) error {
+func init() {
+	imagesCommand.Command = _imagesCommand
+	flags := imagesCommand.Flags()
+	flags.BoolVarP(&imagesCommand.All, "all", "a", false, "Show all images (default hides intermediate images)")
+	flags.BoolVar(&imagesCommand.Digests, "digests", false, "Show digests")
+	flags.StringSliceVarP(&imagesCommand.Filter, "filter", "f", []string{}, "Filter output based on conditions provided (default [])")
+	flags.StringVar(&imagesCommand.Format, "format", "", "Change the output format to JSON or a Go template")
+	flags.BoolVarP(&imagesCommand.Noheading, "noheading", "n", false, "Do not print column headings")
+	// TODO Need to learn how to deal with second name being a string instead of a char.
```

Comment:
Is this still being looked into, or in a future PR?

Reply:
future